File: Course_CS50/pset6/cash/cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed: ")
    if change >= 0:
        break

cash = round(change * 100)

m25 = cash // 25
cash = cash % 25
m10 = cash // 10
cash = cash % 10
m5 = cash // 5
cash = cash % 5

coins = cash + m25 + m10 +m5

print(coins)

File: Course_CS50/pset1/cash.c
//Program count of money

#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do //input coorect date <0
    {
        change = get_float("Change owed: ");
    }
    while(change<=0);

    int cash = round(change * 100);

        int m25 = cash / 25;
        cash = cash % 25;
        int m10 = cash / 10;
        cash = cash % 10;
        int m5 = cash / 5;
        cash = cash % 5;

        int coins = cash + m25 + m10 +m5;
        printf("%i\n", coins);
}
